modules/platforms/dotnet/Apache.Ignite.Tests/FakeServer.cs

```diff
@@ -76,12 +78,15 @@ public FakeServer(bool disableOpsTracking, string nodeName = "fake-server")
         internal FakeServer(
             Func<RequestContext, bool>? shouldDropConnection = null,
             string nodeName = "fake-server",
-            bool disableOpsTracking = false)
+            bool disableOpsTracking = false,
+            int port = 0)
+            : base(port)
         {
             _shouldDropConnection = shouldDropConnection ?? (_ => false);
 
-            Node = new ClusterNode("id-" + nodeName, nodeName, (IPEndPoint)Listener.LocalEndPoint!);
+            Node = new ClusterNode("id-" + nodeName, nodeName, IPEndPoint.Parse("127.0.0.1:" + Port));
```

Comment:
What's wrong with `Listener.LocalEndPoint`?

Reply:
Listener listens on `0.0.0.0` (all interfaces), but client connects to `127.0.0.1`, this breaks assumptions in some tests. Easier to fix here once than to fix multiple tests.